src/directives/directiveResolvers.ts

```diff
@@ -0,0 +1,28 @@
+export const auth = (next: Function) => (root: any, args: any, context: any, info: any) => {
+  if (!context.auth || !context.auth.isAuthenticated()) {
```

Comment:
If context.auth is missing then error should be dofferent. Auth is really bad key as is is very common. Can we do something less common like kauth to avoid overwriting. IMHO it should be configurable

Reply:
Great idea. If we make it configurable, that will be the best option